cypress/e2e/contactForm.cy.js

```diff
@@ -2,10 +2,18 @@ import ContactFormPageObject from '../support/pages/contactForm.pageObject';
 import HomeAndCataloguePageObject
   from '../support/pages/homeСatalogue.pageObject';
 import faker from 'faker';
+import LaptopsPageObject from '../support/pages/laptopsPage.pageObject';
```

Comment:
A product's page has the same structure for each product category so we can name it e.g. `productPageObject.js`.